users/tests/test_user_login_logout.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.test import APITestCase, APIRequestFactory
+from rest_framework import status
+from users.models import User
+from users.tests.factories import UserFactory
+import faker
+
+class LoginTests(APITestCase):
+    def setUp(self):
+        self.user = UserFactory()
+        self.password = faker.Faker().password()
+        self.user.set_password(self.password)
+        self.user.save()
```

Comment:
Check [setUpClass](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass). `setUp` runs before every test, `setUpClass` runs just once before the whole `TestCase`. For your tests creating the user just once seems like a good option.

No need to change it here, just to let you know the option.